ledger/participant-state/src/main/scala/com/daml/ledger/participant/state/v2/CompletionInfo.scala

```diff
@@ -42,13 +43,14 @@ case class CompletionInfo(
     commandId: Ref.CommandId,
     optDeduplicationPeriod: Option[DeduplicationPeriod],
     submissionId: Option[Ref.SubmissionId],
+    statistics: Option[TransactionNodesStatistics],
 ) {
   def changeId: ChangeId = ChangeId(applicationId, commandId, actAs.toSet)
 }
 
 object CompletionInfo {
   implicit val `CompletionInfo to LoggingValue`: ToLoggingValue[CompletionInfo] = {
-    case CompletionInfo(actAs, applicationId, commandId, deduplicationPeriod, submissionId) =>
+    case CompletionInfo(actAs, applicationId, commandId, deduplicationPeriod, submissionId, _) =>
```

Comment:
Should statistics form part of the logging info ?

Reply:
Nope; we don't want to log them on every line. Even this is probably far too much info.

We can log them separately.